arches/app/media/js/views/components/search/map-filter.js

```diff
@@ -274,6 +275,35 @@ define([
                 }
             }, this);
 
+            this.filterByFeatureGeom = function(feature, resourceid=null) {
+                if (feature.geometry.type == 'Point' && this.buffer() == 0) { this.buffer(25); }
+                if (feature.id && resourceid) {
+                    self.searchGeometryFeature({
+                        "featureid": feature.id,
+                        "resourceid": resourceid,
+                        "buffer": {
+                            "width": this.buffer(),
+                            "unit": this.bufferUnit()
+                        },
+                        "inverted": this.filter.inverted()
+                    });
+                }
+                let currentSearchGeoms = self.searchGeometries();
+                this.draw.set({
```

Comment:
This line is giving me a `Uncaught TypeError: cyclic object value for WMS features because the `feature` is also one of the `feature.properties` values. Looking into how to resolve this.

Reply:
in your `map-popup-provider.js` file does it work by setting `feature.properties = {};` ?